includes/demo-poem.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Demo poem.
+ *
+ * @link       https://nataliesmith.dev
+ * @since      1.0.0
+ * @package    Poetwp
+ * @subpackage Poetwp/includes
+ */
+
+/**
+ * Create a demo poem post with Gutenberg blocks and custom fields.
+ *
+ * @return void
+ */
+function poetwp_create_demo_poem() {
+	// Check if the demo poem already exists.
+	$existing_poem = new WP_Query(
+		array(
+			'post_type'      => 'poem',
+			'post_status'    => 'any',
+			'title'          => 'Demo Poem',
+			'posts_per_page' => 1,
+		)
+	);
+
+	if ( ! $existing_poem->have_posts() ) {
```

Comment:
Consider flipping this conditional so that it returns early (guard clause) to avoid basically indenting the rest of the function.

```php
	if ( $existing_poem->have_posts() ) {
		return;
	}

	$poem_content = ...
```

Reply:
Noted #21